android/src/main/java/com/opentokreactnative/utils/Utils.java

```diff
@@ -100,6 +101,25 @@ public static List<IceServer> sanitizeIceServer(ReadableArray serverList) {
         return iceServers;
     }
 
+    public static ArrayList<VideoTransformer> sanitizeVideoTransformerList(PublisherKit publisher, ReadableArray transformerList) {
+        ArrayList<VideoTransformer> nativeVideoTransformers = new ArrayList<>();
+        if (transformerList != null) {
+            for (int i = 0; i < transformerList.size(); i++) {
+                String transformerName = transformerList.getMap(i).getString("name");
+                if (transformerName == "BackgroundReplacement" ) {
+                    // Only implemented in iOS -- ignore in Androd for now.
+                    continue;
+                }
+                VideoTransformer transformer = publisher.new VideoTransformer(
+                    transformerName,
+                    transformerList.getMap(i).getString("properties")
+                );
```

Comment:
Nit: might be simpler to use a `map` since you're taking an "input list" and removing filters that aren't part of your "allowed list".

Reply:
@v-kpheng an you please clarify? The `transformerList` parameter is a ReadableArray (not a map), since the corresponding JavaScript object is an array. And it is an array of objects (in JavaScript), so we are calling `getMap()` here. We don't have an "allowed list" of filters (but we do ignore BackgroundReplacement in Android, for now).

Reply:
NM -- i read too much given we're just ignoring BackgroundReplacement.